### DISCUSSION TITLE
[Feature] Allow adding partner's pictures to albums

### DISCUSSION THREAD

**Discussion by ISibboI** [immich-app/immich#13121] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Currently, at least in the app, I am unable to add pictures to an album if those pictures are shared with me by a partner. I would like to be able to also add pictures from a partner to an album.

In case both partners add the same picture to the same album, that should result in the picture being in the album only once.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Tracked in #12614
